src/main/java/mate/academy/spring/mapper/impl/response/OrderResponseMapper.java

```diff
@@ -0,0 +1,25 @@
+package mate.academy.spring.mapper.impl.response;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.spring.mapper.DtoResponseMapper;
+import mate.academy.spring.model.Order;
+import mate.academy.spring.model.Ticket;
+import mate.academy.spring.model.dto.response.OrderResponseDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class OrderResponseMapper implements DtoResponseMapper<OrderResponseDto, Order> {
+    @Override
+    public OrderResponseDto toDto(Order order) {
+        OrderResponseDto orderResponseDto = new OrderResponseDto();
+        orderResponseDto.setId(order.getId());
+        orderResponseDto.setTimeOfOrder(order.getOrderDate());
+        List<Long> ticketsId = order.getTickets().stream()
```

Comment:
```suggestion
        List<Long> ticketIds = order.getTickets().stream()
```
Fix the same issue in all places